junit-jupiter-api/src/main/java/org/junit/jupiter/api/ClassOrderingStrategy.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.jupiter.api;
+
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+
+import org.apiguardian.api.API;
+
+/**
+ * Supported class ordering strategies.
+ *
+ * @see #DEFAULT
+ * @see #CUSTOM
+ * @since 5.8
+ */
+@API(status = EXPERIMENTAL, since = "5.8")
+public enum ClassOrderingStrategy {
```

Comment:
This class is not necessary in junit-jupiter-api and should be moved to junit-jupiter-engine instead or am I missing sth.?